README.md

```diff
@@ -245,11 +245,17 @@
 ### Term representation
 
 * <a name="underscores-in-numerics"></a>
-  Add underscores to large numeric literals to improve their readability.
+  Add underscores to base-10 numeric literals that have six or more digits.
```

Comment:
I guess if I were writing 65535 port, I'd write it as `65_535` thought (since it still make it more readable). 🤔